package.json

```diff
@@ -48,6 +48,7 @@
     "test:ui:runner": "node scripts/functional_test_runner",
     "test:server": "grunt test:server",
     "test:coverage": "grunt test:coverage",
+    "docs:api": "npm run build:types && api-extractor run && api-documenter markdown -i ./build -o ./docs/development/core/api",
```

Comment:
Yeah, api-extractor looks for `typings` and we use `types` in package.json. Both are valid according to the typescript website. I've submitted https://github.com/Microsoft/web-build-tools/pull/1115 which will fix it, but it's probably a good idea to specify both values in our package.json until it's been published upstream.

Reply:
The upstream change would have written `tsdoc-metadata.json` to the toplevel kibana directory, so I've pushed a commit now that specifies the `tsdocMetadata` location in package.json so that it gets written to `./build`.